helpers/crypto/crypto_test.go

```diff
@@ -1,26 +1,27 @@
 package crypto
 
 import (
-	h "github.com/hacksoc-manchester/www/helpers"
+	r "github.com/hacksoc-manchester/www/helpers/rand"
```

Comment:
How do we decide when to alias a package and when not to? I think we should stick with rand.